lib/string_notifier.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+
+abstract class StringNotifier {
```

Comment:
why abstract

Reply:
Hmm. All methods are protected. StringNotifier should be extended/inherited. I think that will be better to leave abstract in order to indicate it and not allow to create instantion of this class

Reply:
Perhaps will be nice to have something like
```
base class StringNotifier {
```
in flutter